tests/scenarios/RegLegacyTest/config.json

```diff
@@ -21,6 +21,23 @@
           "config": [
             {
               "remediation": [
+                {
+                  "type": "DeletionMarker",
+                  "hive": "HKLM",
+                  "key": "^SYSTEM\\\\.*"
+                },
+                {
+                  "type": "DeletionMarker",
+                  "hive": "HKLM",
+                  "key": "^SOFTWARE.*",
+                  "values": [ "DeletionMarker" ]
+                },
+                {
+                  "type": "DeletionMarker",
+                  "hive": "HKCU",
+                  "key": "^Software.*",
+                  "values": [ "DeletionMarker" ]
```

Comment:
Can you add a test where Values is empty?

Reply:
Sure, I'll add a configuration with empty values, and use existing tests to cover. it.

Reply:
Added.